frontend/src/utils/api.js

```diff
@@ -63,6 +63,34 @@ function getROCurrentYear() {
   // }).then(res => res.json());
 }
 
+function setROActiveYear(readingOlympicsYear) {
+  return {
+    success: true,
+    result: {
+      results: 2019
+    },
+    message: ''
+  };
+  // return fetch(`${API_URL}/setYearOrSomething`, {
+  //   method: 'POST',
+  //   body: JSON.stringify({ year: readingOlympicsYear})
+  // }).then(res => res.json());
+}
+
+function publishAllBooksInYear(readingOlympicsYear) {
+  return {
+    success: true,
+    result: {
+      // idk does it return anything?
```

Comment:
just returns success i believe nothing of interest

Reply:
oops ok